src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```
@@ -645,7 +648,7 @@
     <Compile Include="Microsoft\Data\SqlClient\SqlDelegatedTransaction.cs" />
     <Compile Include="Microsoft\Data\SqlClient\SqlInternalConnection.cs" />
     <Compile Include="Microsoft\Data\SqlClient\SqlInternalConnectionTds.cs" />
-    <Compile Include="Microsoft\Data\SqlClient\SqlTransaction.cs" />
+    <Compile Include="Microsoft\Data\SqlClient\SqlTransaction.cs"/>
```

Revert for consistency
```suggestion
    <Compile Include="Microsoft\Data\SqlClient\SqlTransaction.cs" />
```